src/main/resources/com/ibm/iso8583/Common/ISO8583Format.xsd

```diff
@@ -48,7 +48,7 @@
 	<xs:annotation>
 		<xs:appinfo source="http://www.ogf.org/dfdl/">
 
-	        <dfdl:defineVariable defaultValue="false" external="false" name="secondaryBitmapExists" type="xs:boolean"/>
+	        <dfdl:defineVariable defaultValue="{ fn:false() }" external="false" name="secondaryBitmapExists" type="xs:boolean"/>
```

Comment:
@stevedlawrence Was there a specific reason that this was changed to an expression? The spec allows either.

Reply:
Looks like it's a bug in Daffodil if the spec allows it. Daffodil curently throws an SDE.

Reply:
Yes it is allowed see spec section 7.7.